### PR TITLE
Popover wrapper changes

### PR DESCRIPTION
**Pull request by Cyapow**

### CODE REVIEW

File: styles/overrides/_popover.scss [hunk 1]
```diff
@@ -1,4 +1,6 @@
 .popover.bottom {
```
Comment (kyleharper) [inline styles/overrides/_popover.scss]: Remind me, is the `.bottom` class just for the bottom of the popover (e.g. the content) or is is when the popover is positioned at the bottom of the element you're hovering/clicking on?

File: lib/DropdownMenu/styles.scss [hunk 2]
```diff
@@ -57,7 +57,7 @@ $dropdown-menu-link-padding: 7px 15px !default;
 }
 
 /** Custom **/
-.dropdown__link {
+.dropdown__link, .dropdown-menu a.dropdown__link {
```
Comment (kyleharper) [inline lib/DropdownMenu/styles.scss]: Do we need this declaration to override some styling?

Comment (Cyapow) [reply lib/DropdownMenu/styles.scss]: If we don't the styles get overridden by .dropdown-menu > li > a. Im open to other ways around it though

File: styles/overrides/_popover.scss (hunk 1 above)
Comment (Cyapow) [reply styles/overrides/_popover.scss]: Hmm probably shouldn't use .bottom as its very specific. Should I add an extra class you think and use that?

**Pull request merged**